metals/src/main/scala/scala/meta/metals/providers/HoverProvider.scala

```diff
@@ -27,177 +20,15 @@ object HoverProvider extends LazyLogging {
   ): Hover = {
     val result = for {
       (symbol, _) <- index.findSymbol(uri, line, column)
-      data <- index.data(symbol)
-      tpe <- getPrettyDefinition(symbol, data)
-      document <- index.documentIndex.getDocument(uri)
     } yield {
-      val scalafixIndex = SemanticdbIndex.load(
-        schema.Database(document :: Nil).toDb(None),
-        m.Sourcepath(Nil),
-        m.Classpath(Nil)
-      )
-      val prettyTpe = new TypePrinter()(scalafixIndex).apply(tpe)
+      // TODO: pretty-print SymbolInformation.tpe, blocked by https://github.com/scalameta/scalameta/issues/1479
+      val prettyTpe = symbol.syntax
       Hover(
-        contents = RawMarkedString(language = "scala", value = prettyTpe.syntax) :: Nil,
+        contents = RawMarkedString(language = "scala", value = prettyTpe) :: Nil,
         range = None
       )
     }
     result.getOrElse(Hover(Nil, None))
   }
 
-  /** Returns a definition tree for this symbol signature */
-  private def getPrettyDefinition(
```

Comment:
I'm glad to see this gone in favor of a scalameta pretty-printer!

Reply:
I agree! The current hover output is not helpful but that needs to be fixed upstream.